libs/MotionKit/include/PID.h

```diff
@@ -0,0 +1,37 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <tuple>
+
+#include "interface/drivers/Motor.h"
+
+namespace leka {
+
+class PID
+{
+  public:
+	PID() = default;
+
+	auto processPID([[maybe_unused]] float pitch, [[maybe_unused]] float roll, float yaw)
+		-> std::tuple<float, Rotation>;
+
+  private:
+	const float _Kp			 = 0.3F;
+	const float _Ki			 = 0.0001F;
+	const float _Kd			 = 0.4F;
```

Comment:
Add a comment to say they are found by experience, it will give the information why these are magic numbers and how to modify them if necessary.